sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -331,7 +338,6 @@ class StreamExecution(
 
     val batchTime = (System.nanoTime() - startTime).toDouble / 1000000
     logInfo(s"Completed up to $availableOffsets in ${batchTime}ms")
```

Comment:
> This log line should move to commit

This log contains a local `startTime`. Maybe I can just move codes in `commitBatch` here. What do you think?